server/gameserver.js

```diff
@@ -151,7 +183,8 @@ async function TryReviveServer( request )
 
 	let name = filter.clean( request.query.name )
 	let description = request.query.description == "" ? "" : filter.clean( request.query.description )
-	let newServer = new GameServer( name, description, playerCount, request.query.maxPlayers, request.query.map, request.query.playlist, request.ip, ghost.port, ghost.authPort, request.query.password, modInfo )
+	let region = await getGeoIp( request )
```

Comment:
As I wrote in my PR comment, is this call necessary?
I don't expect servers to change regions between refreshs :)

Reply:
Yeah, it's not necessary then and I'll remove it. I was just unaware of what the function actually does :P

Reply:
Should be fixed with 0e1dc97cd8a20ee31d2aa31d770d2005641cef34

I just switched to re-using ghost's region now.